src/functional/program_processors/codegens/gtfn/itir_to_gtfn_ir.py

```diff
@@ -188,6 +188,8 @@ class GTFN_lowering(eve.NodeTranslator, eve.VisitorWithSymbolTableTrait):
         "and_": "&&",
         "or_": "||",
         "xor_": "^",
+        "mod": "%",
+        "floordiv": "//",
```

Comment:
```suggestion
```
C++ has only one kind of division operator. We have to implement the Python floordiv, by casting (or by introducing a helper function that behaves that way).